pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -205,22 +218,16 @@ func (os *OpenStack) DetachVolume(instanceID, volumeID string) error {
 		return nil
 	}
 
-	if volume.Status != VolumeInUseStatus {
-		return fmt.Errorf("can not detach volume %s, its status is %s", volume.Name, volume.Status)
-	}
```

Comment:
should we check MultiAttach flag ? otherwise, for single attach, we remove that check?

Reply:
Right, this check can be kept as it is. Will update.